compile-release-tools

```diff
@@ -32,11 +32,23 @@ compile() {
   echo "${tool} compiled & installed"
 }
 
+check_deps() {
+  compile github.com/psampaz/go-mod-outdated
+
+  local gmo_args=( '-update' '-direct' )
+  if [ -n "${FAIL_ON_OUTDATED:-}" ]; then
+    gmo_args+=( '-ci' )
+  fi
+
+  go list -u -m -json all | go-mod-outdated "${gmo_args[@]}"
```

Comment:
this is cool!

Reply:
Not yet sure about that yet. It (`go list -m ...`) takes quite a while to check all the deps for potential newer versions. But I think I want to keep it for now, and see how it goes ... We can always easily remove that.

Reply:
er, go-mod-outdated is cool, I just picked this spot to comment on it. `go list ... | foo` is significantly less cool 😉

Reply:
yeah, got it ... I am just sad because `go-mod-outdated` has a dependency on a slow thing, which makes itself less cool 😏

Reply:
yeah, got it ... I am just sad because `go-mod-outdated` has a dependency on a slow thing, which makes itself less cool 😏